overlord/managers_test.go

```diff
@@ -714,6 +786,72 @@ apps:
 	c.Assert(svcFile, testutil.FileContains, "/var/snap/foo/"+revno)
 }
 
+func (ms *mgrsSuite) TestHappyRemoteInstallAndUpgradeWithEpochBump(c *C) {
```

Comment:
we might want a test with more than one snap where one of the updates in mix fails as well